src/styles/variables.scss

```diff
@@ -1,4 +1,4 @@
-$map-height: 500px;
-$map-width: 80%;
+$map-height: 450px;
```

Comment:
Thank you for updating the map size! It looks good and the state information is more eye catching now when a user chooses a state